packages/frontend/core/src/features/auth/login/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const LOGIN_WITH = "username";
+export const LOGIN_NAME = "login";
+export const SIGNUP_NAME = "create account";
+
+export const toTitleCase = (str: string): string => {
+  return str.charAt(0).toUpperCase() + str.slice(1);
+};
```

Comment:
happy with the filename but it means that this should technically be an util and should be moved to a different file - makes it easier to unit test in isolation

Reply:
Okay so this should live at `auth/login/utils/constants.ts` instead?